### ISSUE TITLE
Resource attributes appear as span attributes

### ISSUE THREAD

**Issue by srolel** [metrico/qryn#524]
Hello! I noticed that resource attributes on traces appear under span attributes on every span. Is that by design?

e.g. 

<img width="174" alt="image" src="https://github.com/metrico/qryn/assets/5153007/ef960c56-c5e1-4b90-9fa3-be6a21a9f7ef">

**Comment by akvlad** [comment]
@srolel you're right. For now it's a known design limitation. The resource attributes are inside the span attributes.
